src/containers/intake/utils/IntakeUtils.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import { List, Map } from 'immutable';
+import { DataProcessingUtils } from 'lattice-fabricate';
+import { DataUtils } from 'lattice-utils';
+import { DateTime } from 'luxon';
+import type { UUID } from 'lattice';
+
+import { AppTypes, PropertyTypes } from '../../../core/edm/constants';
+import { EMPTY_VALUE, RoleConstants } from '../../profile/src/constants';
+
+const { VICTIM } = RoleConstants;
+const {
+  DA_CASE,
+  OFFENSE,
+  OFFICERS,
+  PEOPLE,
+  REFERRAL_REQUEST,
+} = AppTypes;
+const {
+  CASE_NUMBER,
+  DATETIME_COMPLETED,
+  DA_CASE_NUMBER,
+  DESCRIPTION,
+  DOB,
+  ETHNICITY,
+  GENERAL_DATETIME,
+  GIVEN_NAME,
+  MIDDLE_NAME,
+  RACE,
+  ROLE,
+  SOURCE,
+  SURNAME,
+} = PropertyTypes;
+const { getEntityAddressKey, getPageSectionKey } = DataProcessingUtils;
+const { getEntityKeyId, getPropertyValue } = DataUtils;
+
+const populateFormData = (
+  person :Map,
+  selectedCaseEKID :?UUID,
+  referralRequest :Map,
+  personCaseNeighborMap :Map,
+  referralRequestNeighborMap :Map
+) :Object => {
+
```

Comment:
thanks! i ended up using your suggestion for the `getPersonData` function to aid in getting the first and second page sections of form data, for which the exact same property values are needed, so makes sense to use a helper. however, i left the logic in `populateFormData` for getting the data in page section three, because that data is grabbed from a variety of entities/properties, and i didn't feel like there was a simpler way to retrieve those values.